addons/website_sale_product_configurator/views/website_sale_product_configurator_templates.xml

```diff
@@ -13,4 +13,9 @@
             <input t-else="" type="hidden" class="d-none js_quantity form-control quantity" name="add_qty" t-att-value="add_qty or 1"/>
         </xpath>
     </template>
+    <template id="optional_product_items_website" inherit_id="sale_product_configurator.optional_product_items">
+        <xpath expr="//t[hasclass('t-optional_product_item')]" position="attributes">
+            <attribute name="t-if" add=" and not combination_info.get('prevent_zero_price_sale', False)" separator=" "/>
+        </xpath>
+    </template>
```

Comment:
Will crash if you update `website_sale_product_configurator` without updating `sale_product_configurator` first.

Reply:
(+ need an exception for adding a template)
How do we proceed, then?

Reply:
```xml
    <template id="optional_product_items_website" inherit_id="sale_product_configurator.optional_product_items">
        <xpath expr="//tr[hasclass('js_product')]" position="replace">
           <t t-if="not combination_info.get('prevent_zero_price_sale', False)"/>
               $0
          </t>
        </xpath>
    </template>
```
?

Reply:
Not working with a replace, but works with a mix of our ideas:
```suggestion
    <template id="optional_product_items_website" inherit_id="sale_product_configurator.optional_product_items">
        <xpath expr="//tr[hasclass('js_product')]" position="attributes">
            <attribute name="t-if">not combination_info.get('prevent_zero_price_sale', False)</attribute>
        </xpath>
    </template>
```